tb_plugin/examples/resnet50_profiler_api.py

```diff
@@ -0,0 +1,59 @@
+import os
+import torch
+import torch.nn as nn
+import torch.backends.cudnn as cudnn
+import torch.optim
+import torch.utils.data
+import torchvision
+import torchvision.transforms as T
+import torchvision.datasets as datasets
+import torchvision.models as models
+
+import torch.profiler
+
+model = models.resnet50(pretrained=True)
+model.cuda()
+cudnn.benchmark = True
+
+transform = T.Compose([T.Resize(256), T.CenterCrop(224), T.ToTensor()])
+trainset = torchvision.datasets.CIFAR10(root='./data', train=True,
+                                        download=True, transform=transform)
+trainloader = torch.utils.data.DataLoader(trainset, batch_size=32,
+                                          shuffle=True, num_workers=0)
+
+criterion = nn.CrossEntropyLoss().cuda()
+optimizer = torch.optim.SGD(model.parameters(), lr=0.001, momentum=0.9)
+device = torch.device("cuda:0")
+model.train()
+
+def output_fn(p):
+    try:
+        os.mkdir("result")
+    except Exception as e:
+        pass
+    p.export_chrome_trace("./result/worker0.pt.trace.json")
+
+with torch.profiler.profile(
+    activities=[
+        torch.profiler.ProfilerActivity.CPU,
+        torch.profiler.ProfilerActivity.CUDA],
+    schedule=torch.profiler.schedule(
+        wait=2,
+        warmup=3,
```

Comment:
Out of curiosity, how did you arrive at this and the above?

Reply:
This script is an example code to generate profiling result(chrome tracing file). The generated file is put to tb_plugin/samples/. 
The first 2 steps are in "wait" state that does not enable CUPTI profiling, the following 3 steps are in "warmup" state that enable CUPTI but does not capture and save the profiling data, the finally 6 steps are in "activate“ state that enable CUPTI and save the profiling data. The finally "p.next_step()" increment the step number.
I'm not sure whether I get your point.

Reply:
I was wondering why you pick warmup=3 rather than say 1 or 2. Did you try different ones and experienced some issue with smaller warmup? And wait = 2 rather than say 10 - is the workload stable after 2 iterations?

Reply:
Well, by my experiment, any wait>=0 and any warmup>=1 could work. Only one warmup is needed, the following steps perform stable on GPU side. Setting to 2 and 3 maybe just give user a feeling that less "special" than setting to 0 and 1. We can add comments here that settings to any wait>=0 and any warmup>=1 are OK.